packages/docusaurus/bin/docusaurus.js

```diff
@@ -55,10 +55,12 @@ program
   });
 
 program
-  .command('eject [siteDir]')
-  .description('copy the default theme into website folder for customization.')
-  .action((siteDir = '.') => {
-    wrapCommand(eject)(path.resolve(siteDir));
+  .command('eject <pluginName> [siteDir] ')
```

Comment:
Maybe we should name it `shadow` :(

Reply:
Shall we add the component name as an option? Sometimes I just want to override one component.